api-authorization/server/authorization-middleware.js

```diff
@@ -0,0 +1,38 @@
+const jwt = require('jsonwebtoken'); // eslint-disable-line
+const ClientError = require('./client-error'); // eslint-disable-line
+
+function authorizationMiddleware(req, res, next) {
+  /* your code here */
+  const token = req.get('X-Access-Token');
+  try {
+    // eslint-disable-next-line no-unused-expressions
+    token;
```

Comment:
I don't understand this line of code. (The lint warning should have given you a clue!)

The expression `token` is just the value of the `token` variable. This line of code does nothing with the value.

It looks like you are trying to test if `token` is defined. That needs to be done in an `if` statement.